workshop/docs/modules/ROOT/pages/creating-a-workbench.adoc

```diff
@@ -32,6 +32,8 @@ image::workbenches/create-workbench-form-image.png[Workbench image, 600]
 +
 image::workbenches/create-workbench-form-size.png[Workbench size, 600]
 
+NOTE: If your cluster displays an "Accelerator" dropdown, choose "None". This workshop does not require the use of a GPU.
```

Comment:
change to:

. If your OpenShift cluster has available GPUs, the *Create workbench* form includes an *Accelerator* option. Select *None*. This {deliverable} does not any GPUs.

Reply:
Yes! Thank you for the correction, @erwangranger